packages/contracts/evm-contracts/contracts/README.md

```diff
@@ -46,4 +46,8 @@ Facilitating monetization
 
 {{Proxy}}
```

Comment:
I think this needs to be deleted (since it was replaced with `Erc20NftSaleProxy` below in this file)

Reply:
Similarly, `NativeProxy` should be removed since we added `NativeNftSaleProxy` instead